api/src/db/models/extract_models.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import BigInteger, Column, ForeignKey, String
+from sqlalchemy.orm import Mapped, mapped_column
+
+from src.adapters.db.type_decorators.postgres_type_decorators import LookupColumn
+from src.constants.lookup_constants import ExtractType
+from src.db.models.base import ApiSchemaTable, TimestampMixin
+from src.db.models.lookup_models import LkExtractType
+
+
+class ExtractMetadata(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "extract_metadata"
+
+    extract_metadata_id = Column(BigInteger, primary_key=True, autoincrement=True)
```

Comment:
Want to use mapped_column, I think I saw Column is being deprecated (already deprecated?) in Sqlalchemy 2.

Autoincrement ends up not needed, a primary key int column ends up as a Serial type which does that automatically.
```suggestion
    extract_metadata_id = mapped_column(BigInteger, primary_key=True)
```